### DISCUSSION TITLE
__about__.py "version" field automatically updated (unwanted behavior)

### DISCUSSION THREAD

**Discussion by daniellepintz** [Lightning-AI/pytorch-lightning#9181] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
When editing code in VSCode, the `pytorch_lightning/__about__.py` file keeps getting automatically updated
```
__version__ = "1.5.0dev" -> __version__ = "20210827"
```

like in https://github.com/PyTorchLightning/pytorch-lightning/pull/8985/commits/5f4f3c5447a30dde3d78b8727a3ca432ec81131b#r697656347

Does anyone know how to stop this from happening? Thanks!

**Comment by carmocca** [comment]
Must be an installed extension

**Reply by daniellepintz** [reply]
thanks!

**Reply by akihironitta** [reply]
@daniellepintz Have you found out what's causing this issue? I'm having the same behaviour in VSCode.

**Reply by daniellepintz** [reply]
Unfortunately I never found out what the issue was :( the problem just kind of went away suddenly... sorry I can't help! :/

**Comment by akihironitta** [comment, marked as answer]
After a few days of my investigation, it turned out that it's not from any extensions but from our script in `.github/`:
https://github.com/PyTorchLightning/pytorch-lightning/blob/83ce1bf5158c4c03f7811e385624bbd903cd9c5f/.github/prepare-nightly_version.py#L13

I locally confirmed that `$ pytest` in the project root directory will run all the scripts matching `*.py` except for those in the excluded directories, and thus it also runs the above file. Not sure why everyone doesn't experience this issue, but I'll submit a fix anyway.

**Reply by daniellepintz** [reply]
that's awesome, thanks!
